hack-kni/verify-commits.sh

```diff
@@ -17,9 +17,13 @@ if [[ "$TRIGGER_BRANCH" == resync-* ]]; then
   exit 0
 fi
 
+if [ -z "$COMMITS" ]; then
```

Comment:
Why no commits detected and no changes detected (in the check below)  aren't the same?

Reply:
actually no good reason, let me fix this

Reply:
Thanks